src/main.rs

```diff
@@ -53,6 +53,8 @@ fn main() {
                  .help("custom corpus directory or artefact files"))
             .arg(Arg::with_name("ARGS").multiple(true).last(true)
                  .help("additional libFuzzer arguments passed to the binary"))
+            .arg(Arg::with_name("TRIPLE").long("target")
```

Comment:
This should be `.takes_value(true)` and default value should also be specified here and not when the value is unwrapped.

/me smacks @Manishearth for merging this.